Signal/src/ViewControllers/AppSettings/AdvancedSettingsTableViewController.m

```diff
@@ -293,8 +293,6 @@ - (void)didToggleThemeSwitch:(UISwitch *)sender
     [Theme setIsDarkThemeEnabled:sender.isOn];
 
     [self updateTableContents];
-
-    // TODO: Notify and refresh.
```

Comment:
We're already notifying using the `theme did change" notification.  AFAIK all relevant views are now observing and handling this event.